### ISSUE TITLE
Transaction data missing an input on Ari

### ISSUE THREAD

**Issue by Pongch** [omisego/elixir-omg#695]
the input is missing in one particular txhash for block height 1000 inside Ari's watcher:

http://quest.ari.omg.network/transaction/0x84b6b6c7cbe14e3432d41389dc3dd0faba9169454dd0e99ad054fb5e17620a9e

transaction data received from the childchain itself do contain single input: 
```
{:ok,
 %OMG.API.State.Transaction.Signed{
   raw_tx: %OMG.API.State.Transaction{
     inputs: [
       %{blknum: 2, oindex: 0, txindex: 0},
       %{blknum: 0, oindex: 0, txindex: 0}, 
       %{blknum: 0, oindex: 0, txindex: 0},
       %{blknum: 0, oindex: 0, txindex: 0}
     ],
     outputs: [
       %{
         amount: 500000000000,
         currency: <<0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
           0>>,
         owner: <<222, 92, 146, 218, 183, 86, 45, 2, 207, 139, 235, 46, 249,
           161, 118, 251, 63, 93, 39, 80>>
       },
       %{
         amount: 9500000000000,
         currency: <<0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
           0>>,
         owner: <<248, 107, 91, 28, 44, 141, 225, 234, 77, 199, 55, 200, 73, 39,
           35, 64, 250, 53, 97, 197>>
       }
```

**Comment by pdobacz** [comment]
UPDATE: Using a recent 112fb91b3199fad2304422ef1e11cea8b32fa3d3 on `v0.1`, this issue does not appear on a locally fully-synced Watcher (synced from scratch):

```
iex(a@user-komp)12> ~c(echo '{"id": "0x84b6b6c7cbe14e3432d41389dc3dd0faba9169454dd0e99ad054fb5e17620a9e"}' | http POST #{watcher}/transaction.get) |>  :os.cmd() |>  Poison.decode!()
%{
  "data" => %{
    "block" => %{...},
    "inputs" => [
      %{
        "amount" => 10000000000000,
        "blknum" => 2, ...
      }
    ],
    "outputs" => [
```

Also, AFAIK we're never deleting DB rows that lead to this information appearing (`ethevents` table), nor doing anything that could cause it to not appear.

Continuing investigation...

**Comment by Pongch** [comment]
would it be worthwhile to explore if there are any more instances of this bug ? cases like this quite easy to filter for IMHO as [0 input N outputs] transaction is not-typical

**Comment by pdobacz** [comment]
root cause fixed in #701, but the problem will still be there until the Ari watcher is resync (which I doubt it will).

In case this solution is fair enough, pls close
